install_config/cluster_metrics.adoc

```diff
@@ -323,15 +323,28 @@ properly.
 ==== Using Generated Self-Signed Certificates
 
 The Metrics Deployer can accept multiple certificates using secrets. If a
-certificate is not passed as a secret, the deployer will generate a self-signed
-certificate to be used instead. For the deployer to generate certificates for
-you, a secret is still required before it can be deployed. In this case, create
-a "dummy" secret that does not specify a certificate value:
+certificate is not passed as a secret, then the deployer will generate and use a
+self-signed certificate instead, however this forces users to accept the certificate as a security exception.
```

Comment:
There's something up with using "however" here, but I can't put my finger on why. Maybe because it usually denotes a negative? I think it might be wise to do away with it and say something like:

"...then the deployer generates a self-signed certificate instead, forcing users to accept the certificate as a security exception."